src/HelixServer.js

```diff
@@ -86,24 +86,23 @@ function executeTemplate(ctx) {
   });
 
   Module._nodeModulePaths = nodeModulePathsFn;
-  const actionArgs = {
+
+  return Promise.resolve(mod.main({
     __ow_headers: owHeaders,
     __ow_method: ctx.method.toLowerCase(),
     __ow_logger: ctx.logger,
     owner: ctx.config.contentRepo.owner,
     repo: ctx.config.contentRepo.repo,
     ref: ctx.config.contentRepo.ref || 'master',
     path: `${ctx.resourcePath}.md`,
+    selector: ctx._selector,
+    extension: ctx._extension,
+    url: ctx._url,
+    params: querystring.stringify(ctx._params),
```

Comment:
The change implies that now `_params` is never `undefined`, old code was testing for it. I would assume there is a test for it which needs to be adjusted then...

Reply:
`querystring.stringify` will return an empty string if `ctx._params` is undefined or an empty object, which is what we test for in [expected_dump.json](https://github.com/adobe/helix-simulator/blob/bf5dd52573b94f80ab907f5b88e9fa622ad3b6c4/test/specs/expected_dump.json#L15)